content/about/notes/1.0.1/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Istio 1.0.1
+weight: 91
+icon: /img/notes.svg
+---
+
+Within this release, we have been addressing some critical issues found by the community when using Istio 1.0.
```

Comment:
Within this release, we have been addressing some critical -> This release address critical issues